client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -82,10 +82,14 @@ interface AcceptDisputeProps {
 /**
  * Disputes and Inquiries have different text for buttons and the modal.
  * They also have different icons and tracks events. This function returns the correct props.
- *
- * @param dispute
  */
-function getAcceptDisputeProps( dispute: Dispute ): AcceptDisputeProps {
+function getAcceptDisputeProps( {
+	dispute,
+	isLoading,
+}: {
+	dispute: Dispute;
+	isLoading: boolean;
+} ): AcceptDisputeProps {
```

Comment:
`getAcceptDisputeProps` is pretty strange to read and reason about. I think this code would be a lot clearer if it just had a dedicated modal for each case (inquiry vs dispute). `AcceptDisputeProps` is a domain-specific interface for a semi-generic modal (e.g. `modalLines`); I don't see a benefit to that over using the React components we have (i.e. a powerful, readable, flexible modal API). 

That said, I don't know if it's safe/worthwhile to refactor this for this change. I would be tempted 😁 

Mentioning because this PR reads as very decoupled – can't really see what changed. Specifically, the button `Submitting…` label should be an implementation detail inside a modal component. I find it strange (and complex) that this is delegated to a function (that now needs a new param).

Reply:
Yes, I agree and I began a refactor of this code in #7339, but it became low-priority and out-of-scope due to time constraints.

I think a refactor should be handled in a separate PR, as it is a much more significant quantity of work and is separate from this PR's goal.